axis_backend/apps/documents/views/document_viewset.py

```
@@ -0,0 +1,511 @@
+            document = self.service.create_document(**serializer.validated_data)
+            response_serializer = self.detail_serializer_class(document)
+            return Response(
+                response_serializer.data,
+                status=status.HTTP_201_CREATED
+            )
+        except ValidationError as e:
+            return Response(
+                {'error': e.message_dict if hasattr(e, 'message_dict') else str(e)},
+                status=status.HTTP_400_BAD_REQUEST
+            )
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)